fabfile.py

```diff
@@ -37,10 +37,6 @@ def install_python_requirements():
         run("pip install --quiet --requirement requirements.txt")
 
 
-def chown_everything():
-    run(f"chown -R {env.user}:www-data {env.path}")
```

Comment:
do we still need to set group `www-data`, though? or are all relevant files set to world-readable?